ports/thestarlitescape/The Starlit Escape.sh

```diff
@@ -45,7 +45,4 @@ $ESUDO chmod +x "$GAMEDIR/gmloader"
 
```

Comment:
```suggestion
pm_platform_helper "$GAMEDIR/gmloader"
```

Reply:
Remove https://github.com/PortsMaster/PortMaster-New/blob/834bea5913202100a4a9d2b3563fb7d175da584d/ports/thestarlitescape/The%20Starlit%20Escape.sh#L40

Replace https://github.com/PortsMaster/PortMaster-New/blob/834bea5913202100a4a9d2b3563fb7d175da584d/ports/thestarlitescape/The%20Starlit%20Escape.sh#L24C28-L24C31 by

`> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1`


Add after https://github.com/PortsMaster/PortMaster-New/blob/834bea5913202100a4a9d2b3563fb7d175da584d/ports/thestarlitescape/The%20Starlit%20Escape.sh#L16C1-L16C22

`[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"`